packages-ui/roosterjs-react/lib/ribbon/plugin/createRibbonPlugin.ts

```diff
@@ -49,6 +50,15 @@ class RibbonPluginImpl implements RibbonPlugin {
      * @param event PluginEvent object
      */
     onPluginEvent(event: PluginEvent) {
+        if (
+            event.eventType == PluginEventType.KeyDown &&
+            event.rawEvent.key == 'k' &&
+            event.rawEvent.ctrlKey
```

Comment:
do you also want to support Meta key on MacOS?

Reply:
And better also check Alt key is not pressed

Reply:
Yes! I can do that, I'll add a check for the Alt key and the Meta key on MacOS.